docs/labwc-config.5.scd

```diff
@@ -206,7 +206,7 @@ this is for compatibility with Openbox.
 
 	*content* defines what the field shows and can be any of:
 
-		- *type* Show view type ("xdg-shell" or "xwayland")
+		- *type* Show view type ("W" for xdg-shell or "X" for xwayland)
```

Comment:
Personally I'd prefer a new content type for this, something like `type_short` (or a better naming).

We try to not surprise users on new releases so changing the look of their setups from one release to the other should not be taken lightly. By defining new types the look doesn't change for existing configs while users can still use the new variations.

Reply:
I had thought to if/else the change to single letter so that the changes would apply to only the all workspaces option. Is that what you're talking about.

Though I find it easier to glance at a single letter instead of trying to parses x..... when they look similar at first glance.

Reply:
> I had thought to if/else the change to single letter so that the changes would apply to only the all workspaces option. Is that what you're talking about.

No, I am suggesting to add a completely new [content type](https://labwc.github.io/labwc-config.5.html#entry_windowswitcher_fields_field_content=_width=%), So it can be configured by the user like
```xml
  <windowSwitcher allWorkspaces="yes">
    <fields>
      <field content="type_compact" width="25%" />
      <field content="trimmed_identifier" width="25%" />
      <field content="title" width="50%" />
    </fields>
  </windowSwitcher>
```
With `type_compact` being the new type.

Reply:
Ok, let me look at what all needs changing for that.

I've made the other changes.